frontend/src/components/product/PriceFilter.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div>
         <!-- Disclosure Component for Price Filter -->
-        <Disclosure defaultOpen>
+        <Disclosure >
```

Comment:
Same here, do we want color filters to start with the default collapse functionality closed?

Reply:
It's correct, in figma design of Caropa you can see the collapse open but It's only to illustrate how it looks when it's open, but this function for default has to keep closed.